go.mod

```diff
@@ -186,14 +172,19 @@ replace (
 	github.com/GoogleCloudPlatform/k8s-cloud-provider => github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534
 	github.com/JeffAshton/win_pdh => github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
 	github.com/MakeNowJust/heredoc => github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
-	github.com/Microsoft/go-winio => github.com/Microsoft/go-winio v0.4.14
-	github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.0.0-20190417211021-672e52e9209d
+	github.com/Microsoft/go-winio => github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
+	github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.8.7
 	github.com/NYTimes/gziphandler => github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
+	github.com/Nvveen/Gotty => github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5
 	github.com/OpenPeeDeeP/depguard => github.com/OpenPeeDeeP/depguard v1.0.1
 	github.com/PuerkitoBio/purell => github.com/PuerkitoBio/purell v1.1.1
 	github.com/PuerkitoBio/urlesc => github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
-	github.com/Rican7/retry => github.com/Rican7/retry v0.1.0
+	github.com/Rican7/retry => github.com/Rican7/retry v0.1.1-0.20160712041035-272ad122d6e5
+	github.com/SeanDolphin/bqschema => github.com/SeanDolphin/bqschema v0.0.0-20150424181127-f92a08f515e1
+	github.com/Shopify/sarama => github.com/Shopify/sarama v1.8.0
+	github.com/Shopify/toxiproxy => github.com/Shopify/toxiproxy v2.1.4+incompatible
```

Comment:
these pull in a lot of additional transitive dependencies and are only used by cadvisor kafka storage integration (which we don't use). we should find a way to isolate storage and container integrations in cadvisor so we don't pull this into our dependency tree if we don't need to